modules/persistence/src/services/metadata/associated_products/index.ts

```diff
@@ -135,21 +133,21 @@ const getAssociatedProducts = async (metadata, sharedMetadataEntry) => {
 
 export const mergeAssociatedToCs = async (metadata) => {
   const { project, branch } = metadata;
-  const sharedMetadata = hasAssociations(metadata) ? metadata : await sharedMetadataEntry(metadata);
+  const umbrellaMetadata = hasAssociations(metadata) ? metadata : await umbrellaMetadataEntry(metadata);
 
   // Short circuit execution here if there's no umbrella product metadata found
-  if (!sharedMetadata) return;
+  if (!umbrellaMetadata) return;
   // Short circuit execution if the project branch is NOT in repo branches
   // If we want to embed with staging builds, then this needs to be turned off
   // or converted so that local metadata ToC is added to tocInsertions
   const isStagingBranch = await !getRepoBranchesEntry(project, branch);
   if (isStagingBranch) return;
 
-  const repoBranchesEntries = await getAllAssociatedRepoBranchesEntries(sharedMetadata);
+  const repoBranchesEntries = await getAllAssociatedRepoBranchesEntries(umbrellaMetadata);
   const repoBranchesMap = mapRepoBranches(repoBranchesEntries);
-  const tocsCursor = await getAssociatedProducts(metadata, sharedMetadata);
+  const tocsCursor = await getAssociatedProducts(metadata, umbrellaMetadata);
```

Comment:
minor. have one ordered param here for getAssociatedProducts

Reply:
Taken care of 💯